lib/pbench/server/database/models/users.py

```diff
@@ -105,7 +105,7 @@ def update(self, **kwargs):
         """Update the current user object with given keyword arguments."""
         try:
             for key, value in kwargs.items():
-                if key == "auth_tokens":
+                if key == "auth_token":
```

Comment:
This feels like a hazard.  The `User` database model (of which this is a method) has a column named `auth_tokens` (well, OK, perhaps it is not a column...it's a `relationship`), and this function accepts a keyword argument of `auth_token` (off by one letter).  This seems like it's ripe for confusion.

In the previous code, the function was intercepting attempts to change `auth_tokens` and handling them specially; in the proposed code, I think we've opened up an avenue for circumventing that interception.

So, if we want to rename the keyword (e.g., at line 108) shouldn't we be renaming the `relationship` in the model (i.e., at line 31) to match?

Reply:
One auth token vs N auth tokens.  A user has potentially N auth tokens.

However, PR #3231 addresses this issue.

Reply:
> One auth token vs N auth tokens. A user has potentially N auth tokens.

I understand the motivation for the change; it's a detail of the execution of the change which concerns me.

> PR #3231 addresses this issue.

What is the expected sequencing -- are we intending #3191 to land before #3231?

Reply:
I was going planning on merging which ever one ran the gauntlet first.  But if folks have a preference I am happy to abide by it.

Either way, I have a heck of merge to deal with. =)

Reply:
> which ever one ran the gauntlet first

This one looks like it has made it....